service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
```

Comment:
I think this got here by accident?

Reply:
This is the second PR I've done this with, I really need to delete those files 🤦